pwiz_tools/Skyline/FileUI/PeptideSearch/ImportPeptideSearchDlg.cs

```diff
@@ -226,6 +226,13 @@ private static void AddPageControl<TControl>(TControl pageControl, TabPage tabPa
             tabPage.Controls.Add(pageControl);
         }
 
+        private UserControl GetPageControl(TabPage tabPage)
+        {
+            var wizardPageControls = tabPage.Controls.OfType<UserControl>().ToArray();

```

Comment:
Could also be First() if asserting there are not 2 is not important. First will throw if their are none, just like your Assume.

Reply:
BTW. If you want to throw an error if the number of elements is anything other than "1" you can use "Single()":
`return tabPage.Controls.OfType<UserControl>().Single();`

Reply:
I was aware of the Linq options here but I thought an explicit Assume was more self-documenting (obviously not enough). I'm making an assumption here that the tabPage only has one UserControl and it should be an error if there isn't.

Reply:
I went with `Single()` and a comment in the code about the assumption.